lib/ansible/modules/system/aix_chsec.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Christian Tremel (@flynn1973)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: aix_chsec
+short_description: Modify AIX stanza files
+version_added: '2.8'
+description:
+- Modify stanzas to AIX config files using the C(chsec) command.
+options:
+  path:
+    description:
+    - Path to the stanza file.
+    type: path
+    required: true
+    aliases: [ dest ]
+  stanza:
+    description:
+    - Name of stanza.
+    type: str
+    required: true
+  options:
+     description:
+     - A list of key/value pairs, e.g. C(key=val,key=val).
+     type: list
+  state:
+     description:
+     - If set to C(absent) the whole stanza incl. all given options will be removed.
+     - If set to C(present) stanza incl.options will be added.
+     - To remove an option from the stanza set to C(present) and set key to an empty value (key=).
+     - All rules, allowed file-stanza combos or allowed files for the C(chsec) command also applies here.
+     type: str
+     choices: [ absent, present ]
+     default: present
+seealso:
+- name: The chsec manual page from the IBM Knowledge Center
+  description: Changes the attributes in the security stanza files.
+  link: https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/com.ibm.aix.cmds1/chsec.htm
+author:
+- Christian Tremel (@flynn1973)
+'''
+
+EXAMPLES = r'''
+- name: Add an LDAP user stanza
+  aix_chsec:
+    path: /etc/security/user
+    stanza: ldapuser
+    options: SYSTEM=LDAP,registry=LDAP
+    state: present
+    mode: '0644'
+
+- name: Change login times for user
+  aix_chsec:
+    path: /etc/security/user
+    stanza: ldapuser
+    options: logintimes=:0800-1700
+    state: present
+
+- name: Remove registry option from stanza
+  aix_chsec:
+    path: /etc/security/user
+    stanza: ldapuser
+    options:
+    - SYSTEM=LDAP
```

Comment:
To me it seems more logical if the format is not a list, but a dictionary.
```yaml
      options:
        SYSTEM: LDAP
        registry: LDAP
        logintimes: :0800-1700
```

The only reasons for using a list here, would be to ensure a specific order for processing, or to ensure that a specific key can be duplicate (so provided keys are not unique). But I don't think these are no applicable here.

Reply:
If you are worried about breaking backward compatibility with existing users, you can use `type: raw` and test if a string, a list or a dict was provided, and handle strings as `key=val`, use dicts as an iterable `key: value` store, and a list could contain either of those two.

And then only document/showcase the dicts syntax, and provide a deprecation message for people using a list or a string. Which could then be removed after 4 major releases (which is what we typically do).